src/pages/givingform.tsx

```diff
@@ -11,39 +11,93 @@ import Layout from "../components/layout"
-    regularGiftFrequency: "2138441357",
-    regularGiftCommencementDate: "825676099",
-    thisGiftIsEligibleForGiftAid: "567316447",
-    allFutureGiftsAreEligibleForGiftAid: "518456171",
-    retrospectivelyReclaimGiftAid: "1656623006",
-    retrospectiveGiftAidClaimStartDate: "546486386",
+type FormConfig = {
+    googleFormId: string
+    fieldNameToEntryId: {
+        givenName: string
+        familyName: string
+        streetAddress: string
+        extraAddressLine: string
+        townCity: string
+        postCode: string
+        telephoneNumber: string
+        emailAddress: string
+        giftAmountInGBP: string
+        giftType: string
+        regularGiftFrequency: string
+        regularGiftCommencementDate: string
+        thisGiftIsEligibleForGiftAid: string
+        allFutureGiftsAreEligibleForGiftAid: string
+        retrospectivelyReclaimGiftAid: string
+        retrospectiveGiftAidClaimStartDate: string
+    }
+    warning: ReactElement | null
+}
+
+const config: { [configName: string]: FormConfig } = {
+    production: {
+        googleFormId:
+            "1FAIpQLSeED6ffZVCYFG5amGWUtTGWkr8EI-rbPO2i_f-z0Ur6XvTNTw",
+        fieldNameToEntryId: {
+            givenName: "403087895",
+            familyName: "383939029",
+            streetAddress: "2131927972",
+            extraAddressLine: "54229700",
+            townCity: "1247583245",
+            postCode: "524386314",
+            telephoneNumber: "899696842",
+            emailAddress: "1518406431",
+            giftAmountInGBP: "710981495",
+            giftType: "1615869123",
+            regularGiftFrequency: "2138441357",
+            regularGiftCommencementDate: "825676099",
+            thisGiftIsEligibleForGiftAid: "567316447",
+            allFutureGiftsAreEligibleForGiftAid: "518456171",
+            retrospectivelyReclaimGiftAid: "1656623006",
+            retrospectiveGiftAidClaimStartDate: "546486386",
+        },
+        warning: null,
+    },
+    development: {
+        googleFormId:
+            "1FAIpQLSc5aNC3Kfax5kWBRTzQrhuJBehMrQPnwkugkv17k3MeMNtfcQ",
+        fieldNameToEntryId: {
+            givenName: "1341689296",
+            familyName: "2098486112",
+            streetAddress: "1905441603",
+            extraAddressLine: "517339018",
+            townCity: "300162044",
+            postCode: "1869992928",
+            telephoneNumber: "1892238525",
+            emailAddress: "163193888",
+            giftAmountInGBP: "1940242436",
+            giftType: "482226407",
+            regularGiftFrequency: "876322395",
+            regularGiftCommencementDate: "1395887961",
+            thisGiftIsEligibleForGiftAid: "1683708464",
+            allFutureGiftsAreEligibleForGiftAid: "1965645277",
+            retrospectivelyReclaimGiftAid: "633586953",
+            retrospectiveGiftAidClaimStartDate: "1572694217",
+        },
+        warning: devWarning,
+    },
 }
 
+const configName = process.env.GIVING_FORM_CONFIG ?? "production"
```

Comment:
I'd be tempted to just throw an error if this isn't defined – better to be explicit than defaulting to prod?

Reply:
good shout.

Reply:
Sorted.